### PR TITLE
Correct filter in ExecutionView.kt

### PR DESCRIPTION
**Pull request by Arrgentum**
## Whats Added:
* Filter value status and testSuite now changes the values of the output table data
* Safe value status and testSuite for future use
* Some corrected README.md

## Issue:
first part #838

### CODE REVIEW

**Review by sanyavertolet — APPROVED** [review]
lgtm

**Review by petertrr — CHANGES_REQUESTED** [review]
See comment above

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/TableComponent.kt [hunk 1]
```diff
@@ -70,6 +71,16 @@ external interface TableProps<D : Any> : Props {
      * Lambda to update number of pages
      */
     var getPageCount: (suspend (pageSize: Int) -> Int)?
+
+    /**
+     * Test Result Status to filter by
+     */
+    var status: TestResultStatus?
```
[the remark this answers is not in the document]
Comment (Arrgentum) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/TableComponent.kt:78]: If I understand correctly, then status and testSuite are property in tableProps

[the remark this answers is not in the document]
Comment (petertrr) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/tables/TableComponent.kt:78]: Ok, we'll need a way to inherit `TableProps` and override dependencies, if needed

**Review by petertrr — APPROVED** [review]

**Pull request merged**